content/id/docs/concepts/extend-kubernetes/extend-cluster.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Memperluas Kubernetes Cluster Anda
```

Comment:
```suggestion
title: Memperluas Kluster Kubernetes Kamu
```

We should use "Kamu" instead to be more friendly and welcoming to the readers.
We follow the Kubernetes' guidelines.

Reply:
done